src/components/sections/Faq.tsx

```diff
@@ -110,6 +110,15 @@ export const faqs = slugifyFaqs([
       </p>
     ),
   },
+  {
+    title: "How do I coordinate with folks?",
+    body: (
+      <p>
+        The best way to communicate is to join our slack <a href='https://turnonthefawcetts.slack.com/join/invite/enQtMjQ1MDMzNjY5MDkxOS05OWFhMzAxZDkwZDFlYzJiZmFmODZmNDg1ZWE4YWE0MTg1MzcyMDBlNmEwZDY4ZTgzNDAwZTA3MjFhYTgyMDYz#/credentials'>here!</a>
```

Comment:
Shit, is there any way to put this under the protected path john set up at  https://turnonthefawcetts.com/mvps  ? 

I don't want everyone having that link

Reply:
Sorry that was not in reqs my bad